pkgs/development/libraries/cegui/default.nix

```diff
@@ -47,6 +47,7 @@ stdenv.mkDerivation {
 
   passthru.updateScript = unstableGitUpdater {
     branch = "v0";
+    hardcodeZeroVersion = true; # tags have format "vx-y-z", updater lacks logic for converting to dot-separated
```

Comment:
Same as `chirp`: Might need post-updater code to convert `x-y-z-unstable-YYYY-MM-DD` into `x.y.z-unstable-YYYY-MM-DD`.